src/main/java/bio/terra/service/policy/PolicyService.java

```diff
@@ -119,14 +107,14 @@ public RepositoryStatusModelSystems status() {
       publicApi.getStatus();
       return new RepositoryStatusModelSystems()
           .ok(true)
-          .critical(policyServiceConfiguration.enabled())
+          .critical(true)
```

Comment:
It looks like the convention, for services whose `critical` status isn't configurable, is to put this setting in the caller. See https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/status/StatusService.java#L48